katello-host-tools/katello-host-tools.spec

```diff
@@ -1,7 +1,7 @@
-Name: katello-agent
+Name: katello-host-tools
 Version: 3.0.0
-Release: 2%{?dist}
-Summary: The Katello Agent
+Release: 3%{?dist}
+Summary: A set of commands and yum plugins that support a Katello host 
 Group:   Development/Languages
 License: LGPLv2
 URL:     https://github.com/Katello/katello-agent
```

Comment:
If this gets merged in the next ~24 hours, i'll do a followup pr.  Otherwise I'll update before merging.